homeassistant/components/habitica/icons.json

```diff
@@ -1,5 +1,51 @@
 {
   "services": {
     "api_call": "mdi:console"
+  },
+  "entity": {
```

Comment:
Personal thing, can you add the services to the bottom? :p

Reply:
sure :)